tools/cldr-apps/src/main/webapp/js/new/cldrGear.js

```diff
@@ -6,6 +6,13 @@
  *
  * Use an IIFE pattern to create a namespace for the public functions,
  * and to hide everything else, minimizing global scope pollution.
+ *
+ * TODO: "Avoid dropdown menus": "Usability studies show that dropdown menus are annoying";
+ * "difficult for search engines to crawl" https://www.orbitmedia.com/blog/website-navigation/
+ * Survey Tool's current dropdown gear menu is also problematic in terms of library dependencies,
+ * invalid html, inability to scroll (e.g., when console log is open and screen is small),
+ * and failure to close immediately when some items are clicked on. It would be cleaner to
+ * open a new special page ("#gear"), with a list of items, when the gear menu is clicked on.
```

Comment:
@btangmu could be good, and now that the gear is a module it should be simple..

Reply:
yes